### ISSUE TITLE
TypeScript types are wrong when `module` is `node16`

### ISSUE THREAD

**Issue by dstaley** [open-cli-tools/concurrently#494]
When using `concurrently` in a project with `module` set to `node16` (or `nodeNext`) in `tsconfig.json`, the default export of `concurrently` is not correct:

```ts
import concurrently from 'concurrently'
concurrently(); // This expression is not callable. Type 'typeof import("node_modules/concurrently/dist/src/index")' has no call signatures.
```

I believe this is due to the fact that the `types` field points to `dist/src/index.d.ts` for both the `require` and `import` fields in `exports`, which is not going to work since the files are two different module systems. You can see more compatibility issues [here](https://arethetypeswrong.github.io/?p=concurrently%408.2.2).

I'm happy to send in a PR to fix this in a few days, but I just wanted to throw the issue out there for awareness.

**Comment by gustavohenke** [comment]
Hey! I believe this might've been fixed by #456, it's just that it hasn't been released yet.
It's been a while since I played with the type definitions, so at least https://arethetypeswrong.github.io gives all ✅ with the output of `npm pack`.

**Comment by dstaley** [comment]
oh you're totally right! it looks like that PR should solve the issue. sorry for the noise!

**Comment by gustavohenke** [comment]
Hey! This is now fixed in v9.0.0.
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
